src/python/requirements-system.txt

```diff
@@ -1,6 +1,6 @@
-streamlit==1.12.0
-protobuf==3.18
-graphistry==0.27.1
+streamlit==1.21.0
+protobuf==3.20.3
```

Comment:
updating protobuf generally breaks rapids

Reply:
This was an error I was getting which made me decide to upgrade.. Do you think there could be other workarounds? 
What else can I do to test that rapids is working properly?

```
ImportError: cannot import name 'builder' from 'google.protobuf.internal' (/opt/conda/envs/rapids/lib/python3.8/site-packages/google/protobuf/internal/__init__.py)
Traceback:
File "/apps/views/demo_avr/__init__.py", line 308, in run_all
    graph_url = run_filters(splunk_edf) #**sidebar_filters)
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/streamlit/runtime/caching/cache_utils.py", line 194, in wrapper
    return cached_func(*args, **kwargs)
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/streamlit/runtime/caching/cache_utils.py", line 223, in __call__
    return self._get_or_create_cached_value(args, kwargs)
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/streamlit/runtime/caching/cache_utils.py", line 248, in _get_or_create_cached_value
    return self._handle_cache_miss(cache, value_key, func_args, func_kwargs)
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/streamlit/runtime/caching/cache_utils.py", line 302, in _handle_cache_miss
    computed_value = self._info.func(*func_args, **func_kwargs)
File "/apps/views/demo_avr/__init__.py", line 263, in run_filters
    graph_url = marlowe.umap(render=False)
File "/apps/views/demo_avr/marlowe.py", line 218, in umap
    self.g = graphistry.nodes(self.edf).umap(
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/graphistry/umap_utils.py", line 457, in umap
    res.umap_lazy_init(engine=engine, suffix=suffix)
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/graphistry/umap_utils.py", line 181, in umap_lazy_init
    engine_resolved = resolve_umap_engine(engine)
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/graphistry/umap_utils.py", line 86, in resolve_umap_engine
    has_cuml_dependancy_, _, cuml = lazy_cuml_import_has_dependancy()
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/graphistry/umap_utils.py", line 42, in lazy_cuml_import_has_dependancy
    import cuml  # type: ignore
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/cuml/__init__.py", line 17, in <module>
    from cuml.common.base import Base
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/cuml/common/__init__.py", line 17, in <module>
    from cuml.common.array import CumlArray
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/cuml/common/array.py", line 26, in <module>
    from cudf import DataFrame
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/cudf/__init__.py", line 70, in <module>
    from cudf.io import (
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/cudf/io/__init__.py", line 8, in <module>
    from cudf.io.orc import read_orc, read_orc_metadata, to_orc
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/cudf/io/orc.py", line 14, in <module>
    from cudf.utils.metadata import (  # type: ignore
File "/opt/conda/envs/rapids/lib/python3.8/site-packages/cudf/utils/metadata/orc_column_statistics_pb2.py", line 7, in <module>
    from google.protobuf.internal import builder as _builder
```

Reply:
Can you delete ~all dependencies that may be changing library versions of things and see if still broken?

I wonder if it's graphistry[ai], which can be worked around by using a more modern (graphistry) base image for the dockerfile, any of the 2.40.x

I'm almost nearing the point of testing a new base image with latest pygraphistry, just working through landing percy's pr first...